policyengine_us/parameters/gov/irs/credits/lump_sum_distribution/expected_remaining_years_of_life.yaml

```diff
@@ -0,0 +1,488 @@
+description: The IRS provides the following multiples corresponding to the filer's expected remaining years of life. 
+metadata:
+  type: single_amount 
+  threshold_unit: year
+  period: year
+  amount_unit: /1
+  label: Expected remaining years of life
+  reference:
+      - title: Actuarial Values 3A Publication 1457 (Rev. 5-2009)
+        href: https://www.irs.gov/retirement-plans/actuarial-tables
```

Comment:
Be more specific - which file URL and page number. Are there multiple files for different years?